src/ethtool/Makefile

```diff
@@ -11,6 +11,10 @@ $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	git clone https://salsa.debian.org/kernel-team/ethtool/
 	pushd ethtool
 	git checkout tags/debian/1%$(ETHTOOL_VERSION_BASE)-1
+ifneq (,(filter $(CONFIGURED_ARCH),arm64 armhf))
+	# Disabling Tests
+	sed -i -e 's/\(override_dh_auto_configure:\)/override_dh_auto_test:\n\techo \"Skipping Tests\"\n\n\1/g' debian/rules
```

Comment:
NO NEED TO IT HERE.

you can refer this example.

rules/sairedis.mk:$(LIBSAIREDIS)_DEB_BUILD_OPTIONS = nocheck

it will disable check

Reply:
Thanks for the example, @lguohan